### ISSUE TITLE
iD should show "area presets" instead of "linear presets" for untagged closedways included in the multipolygon as inner

### ISSUE THREAD

**Issue by serhii-muchychka** [openstreetmap/iD#9672]
### Description

For example, let's take a forest that contains such untagged inner ring. Most likely, it will be either grass or a scrub. But iD offers a linear presets, such as a road, a river, ..., which looks less logical

**Comment by bhousel** [comment]
Did you know you can just add an `area=yes` tag manually - then you'll be offered the Area presets?

**Comment by serhii-muchychka** [comment]
Yes, I know. But it is not convenient. I highlighted that it could be improved. iD developers can decide if this is necessary

**Comment by tyrasd** [comment]
For closed ways that are an inner member of a relation, it is indeed more likely that they should be tagged as a polygon (`barrier=fence` would be one counter example).

There is even the bigger question: shouldn't all closed ways offer also area presets (and vice versa) in order to quickly change between different feature types?

**Comment by tordans** [comment]
> There is even the bigger question: shouldn't all closed ways offer also area presets (and vice versa) in order to quickly change between different feature types?

Changing the feature type is quite painful ATM. AFAIK the only way is to manually add `area=yes` to the line. This comes up when splitting an area into separate areas, for example. If the UI "does not break" (AKA I still understand the preset selection list"), I am very much in favor of listing presets for both geometries in this case.

**Comment by jjiglesiasg** [comment]
could it be developed a shortcut that convert a line closed loop in an area? whatever is part of a relation or not? for mapping adjacent landuse or natural where you need to connect different nodes, is much easier to do it using the line drawing mode than the area drawing mode
